ui/components/component-library/form-text-field/form-text-field.scss

```diff
@@ -0,0 +1,9 @@
+.mm-form-text-field {
+  --help-text-opacity-disabled: 0.5;
+
+  &--disabled {
+    .mm-form-text-field__help-text {
+      opacity: var(--help-text-opacity-disabled);
```

Comment:
Not sure about adding opacity to the help text. @garrettbear or @brianacnguyen do you have any opinions on this? Could we ever want the help text to be normal opacity that describes why the input is disabled?

Normal

<img width="1185" alt="Screen Shot 2022-11-14 at 6 19 06 PM" src="https://user-images.githubusercontent.com/8112138/201795234-3965f8f5-e07c-4803-8183-03745a339cf7.png">

Disabled

<img width="1440" alt="Screen Shot 2022-11-14 at 6 18 34 PM" src="https://user-images.githubusercontent.com/8112138/201795263-681b0a9e-de02-4d12-b1bb-06f5186153d1.png">


cc @Akatori-Design

Reply:
OK thanks! Let's leave it as is for now 🙏